src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt

```diff
@@ -255,5 +246,6 @@ tasks.register('dist') {
 if(enableGraalNative == 'true') {
   apply from: file("nativeimage.gradle")
 }
-"""
+
+""".trimIndent()
```

Comment:
Lets us not worry about the first and last line, so we can leave the first line empty and have them [correctly aligned](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-indent.html)

Reply:
Cool... I don't fully understand how Kotlin handles this, but reading this should help me get a grip on it.

Reply:
This might not work as well as I thought it did. The ktlint task on GH Actions reports:
```
/home/runner/work/gdx-liftoff/gdx-liftoff/src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt:250:1 Unexpected indent of multiline string closing quotes

```

So apparently ktlint considers this poor style. No idea why.

Reply:
The build won't complete unless ktlint finishes, so if possible, run the `ktlintFormat` task to find out what it doesn't like and have it try to fix the problems.

Reply:
[ade7a63](https://github.com/libgdx/gdx-liftoff/pull/155/commits/ade7a635bb61f90113cfbbeaa00db635aafa3f06) whoops, now it should be good to go